README.md

```diff
@@ -151,72 +156,28 @@ Start all boulder components with test configs (Ctrl-C kills all):
 
     ./start.py
 
-Run tests:
+Run tests (this requires that the services are not already running as it will start them itself):
```

Comment:
At one point test.sh was supposed to work smoothly if the services were already running, to save startup overhead. Is that no longer the case?